variables.tf

```diff
@@ -0,0 +1,134 @@
+variable "additional_iam_policy_arns" {
+  default     = []
+  description = "List of ARN policies for addional builder permissions"
+  type        = list(string)
+}
+
+variable "description" {
+  default     = null
+  description = "description of component"
+  type        = string
+}
+
+variable "instance_types" {
+  default     = ["t3.medium"]
+  description = "Instance types to create images from. It's unclear why this is a list. Possibly because different types can result in different images (like ARM instances)"
+  type        = list(string)
+}
+
+variable "key_pair" {
+  default     = null
+  description = "EC2 key pair to add to the default user on the builder"
+  type        = string
+}
+
+variable "license_config_arns" {
+  default     = null
+  description = "If you're using License Manager, your ARNs go here"
+  type        = list(string)
+}
+
+variable "log_bucket" {
+  default     = null
+  description = "Bucket to store logs in. If this is ommited logs will not be stored"
+  type        = string
+}
+
+variable "log_prefix" {
+  default     = null
+  description = "S3 prefix to store logs at. Recommended if sharing bucket with other pipelines"
+  type        = string
+}
+
+variable "name" {
+  description = "name to use for component"
+  type        = string
+}
+
+variable "recipe_arn" {
+  description = "ARN of the recipe to use. Must change with Recipe version"
+  type        = string
+}
+
+variable "regions" {
+  default = [
+    "us-east-1",
+    "us-east-2",
+    "us-west-1",
+    "us-west-2",
+    "ca-central-1"
+  ]
+}
+
+variable "schedule" {
+  default = {
```

Comment:
does it make sense to have a default?

Reply:
Specifically want a default for `PipelineExecutionStartCondition`. I _could_ split this into two different vars, but I also think this cron is a decent default and reduces the things you're required to think about when using this module.